### PR TITLE
Update customer-key-manage.md

### PR DESCRIPTION
**Pull request by sathana87**

### CODE REVIEW

File: microsoft-365/compliance/customer-key-manage.md [hunk 1]
```diff
@@ -404,19 +404,7 @@ To initiate the data purge path, complete these steps:
...
-   You do not delete the Azure Key Vault for SharePoint Online. Key vaults may be shared among several SharePoint Online tenants and DEPs.
-
-2. Contact Microsoft to delete the availability key.
-
-    When you contact Microsoft to delete the availability key, we'll send you a legal document. The person in your organization who signed up as an approver in the FastTrack offer during onboarding needs to sign this document. Normally, this is an executive or other designated person in your company who's legally authorized to sign the paperwork on behalf of your organization.
-
-3. Once your representative signs the legal document, return it to Microsoft (usually through an eDoc signature).
-
-   Once Microsoft receives the legal document, we run cmdlets to trigger the data purge which performs crypto deletion of the tenant key, site key, and all individual per-document keys, irrevocably breaking the key hierarchy. Once the data purge cmdlets complete, your data has been purged.
+Sharepoint doesnot support offboarding for availability key. Instead delete the primary key vault.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-key-manage.md:407]:
```suggestion
SharePoint does not support offboarding for the availability key. Instead, delete the primary key vault. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @sathana87, kindly commit the suggested change. Many thanks.

**Comment by yogkumgit** [comment]
@sathana87 Please implement the suggested changes, in the event of no response we may may have to close this PR. Thanks

**Comment by yogkumgit** [comment]
@sathana87 We appreciate any feedback that improves the content in the Microsoft docs but we haven't had any response from you so we are going to close this issue.  
Please feel free to re-open this issue if you see an improvement is required in the specific area of this documentation. We will certainly update this documentation with that valuable information. Thanks again, for taking out some time to open the issue. Appreciate and encourage you to do the same in future also.

**Pull request closed without merging**